packages/react/src/components/accordion/accordion-styles.ts

```diff
@@ -0,0 +1,96 @@
+import styled, { css } from 'styled-components';
+import { Theme } from '../../themes';
+
+type StyledFunction = (isExpanded: boolean, theme: Theme) => ReturnType<typeof css>;
+type AccordionSectionFunction = (theme: Theme) => ReturnType<typeof css>;
+
+export const StyledAccordionGroup = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+`;
+
+export const AccordionSectionStyled: AccordionSectionFunction = (theme) => css`
+    background: ${theme.greys['neutral-02']};
+    border: 0 solid ${theme.greys['neutral-15']};
+    border-radius: 0 0 var(--border-radius-2x) var(--border-radius-2x);
+    border-top-width: 0;
+    margin-bottom: var(--spacing-1x);
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.5s ease, border 0.5s ease;
+    &.expanded {
+        border: 1px solid ${theme.greys['neutral-15']};
+        border-top-width: 0;
+        max-height: 500px;
+        transform: translateZ(0);
+        transition: max-height 1s ease, border 1s ease;
+        will-change: max-height, border;
+    }
+`;
+
+export const AccordionBodyStyled: AccordionSectionFunction = (theme) => css`
+    background: ${theme.greys['colored-white']};
+    color: ${theme.greys['neutral-90']};
+    font-size: 0.75rem;
+    font-weight: var(--font-normal);
+    letter-spacing: 0.015rem;
+    line-height: 1.7;
+    padding: var(--spacing-2x) var(--spacing-3x) var(--spacing-3x) var(--spacing-5x);
+`;
+
+export const HeadingStyled = css`
+    position: relative;
+`;
+
+export const ButtonStyled: StyledFunction = (isExpanded, theme) => css`
+    align-items: flex-start;
+    border: 1px solid ${theme.greys['neutral-15']};
+    border-radius: ${isExpanded ? 'var(--border-radius-2x) var(--border-radius-2x) 0 0' : 'var(--border-radius-2x)'};
+    color: ${theme.greys['neutral-90']};
+    font-size: 0.875rem;
+    font-weight: var(--font-normal);
+    justify-content: start;
+    letter-spacing: 0.015rem;
+    line-height: 1.5;
+    min-height: var(--spacing-5x); /* TODO change space when updating thematization */
+    padding: var(--spacing-1x); /* TODO change space when updating thematization */
+    text-align: left;
+    text-transform: none;
+    transition: border-radius 0.2s ease;
+    width: 100%;
+
+    &[aria-expanded='true'] {/* TODO change colors when updating thematization */
+        background: ${theme.greys.white};
+        color: ${theme.greys['neutral-90']};
+        transition: border-radius 0.2s ease;
```

Comment:
The radius animation is a bit weird when collapsing, but I'm not sure if there's something we can do.

Reply:
added delay to improve transition on collapsing. hope that will look better.